### PR TITLE
Fix TODO-s in README

### PR DESCRIPTION
**Pull request by szepeviktor**
Those brackets need an empty space for incomplete items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<p align="center"><a href="https://johndatserakis.github.io/vue-navigation-bar/" target="_blank" rel="noopener noreferrer"><img src="./src/assets/images/lockup-color.png" width="200" style="max-height: 100px; " alt="vue-navigation-bar"></a></p>
```
Comment (szepeviktor) [inline README.md:1]: And a stray space here.

**Comment by szepeviktor** [comment]
BTW TODO-s like to live in GitHub Issues.

[_source_](https://github.com/szepeviktor/debian-server-tools/blob/df839fb3f58253a33e45c519153e056e9a91b96b/webserver/Continuous-integration-Continuous-delivery.yml#L220-L224)

**Comment by johndatserakis** [comment]
Thank you.

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**